timescaledb/getting-started/add-data.md

```diff
@@ -1,53 +1,63 @@
 
-For each city, we record the following metrics:
 ```bash
-* time: Timestamp of data calculation
-* timezone: Shift in seconds from UTC
-* city_name: City name
-* temp_c: Temperature in degrees celsius
-* feels_like_c:  This temperature parameter accounts for the human perception of weather
-* temp_min_c: Minimum temperature
-* temp_max_c Maximum temperature
-* pressure_hpa: Atmospheric pressure (at sea level) in hPa
-* humidity_percent:  Humidity as a percentage of maximum water vapor
-* wind_speed_ms: Wind speed in meters per second
-* wind_deg: Wind direction, degrees (meteorological)
-* rain_1h: Rain volume for the last hour, mm
-* rain_3h: Rain volume for the last 3 hours, mm
-* snow_1h: Snow volume for the last hour, in mm
-* snow_3h: Snow volume for the last 3 hours, in mm
-* clouds_percent: Cloudiness as a percentage
-* weather_id: Weather condition id
+    * time: timestamp column incrementing at a tick level (second by second)
+    * symbol: symbols representing a company, mappings found in the `company` table
+    * price: stock quote price for a company at the given timestamp
+    * day_volume: how many shares are traded each day, NULL values indicate the market is closed
 ```
 
-## Accessing the dataset
+`company`: mapping for symbols to company names
 
-We provide a CSV file with commands for inserting the data into your TimescaleDB instance.
+```bash
+    * symbol: the symbol representing a company name
+    * name: corresponding company name
+```
 
-Download the CSV file (in ZIP format) below and insert it into your database from psql.
+<procedure>
 
-Download CSV: <tag type="download">[weather_data.zip](https://s3.amazonaws.com/assets.timescale.com/docs/downloads/weather_data.zip)</tag>
+### Accessing the dataset
 
-After unzipping the file, use the following command (which assumes `weather_data.csv` is located in your current directory):
+1.  Before ingesting the data, you must create another table, `company`. Run the 
+    following SQL to do this. 
 
-```sql
--- copy data from weather_data.csv into weather_metrics
-\copy weather_metrics (time, timezone_shift, city_name, temp_c, feels_like_c, temp_min_c, temp_max_c, pressure_hpa, humidity_percent, wind_speed_ms, wind_deg, rain_1h_mm, rain_3h_mm, snow_1h_mm, snow_3h_mm, clouds_percent, weather_type_id) from './weather_data.csv' CSV HEADER;
-```
-Now that you’re up and running with historical data inside TimescaleDB and a
-method to ingest the latest data into your database, let’s start querying the data.
+    ```sql
+    CREATE TABLE IF NOT EXISTS company (
+        symbol text NOT NULL,
+        name text NOT NULL
+    );
+    ```
+
+1.  Download the following CSV files (in ZIP format). There are two files with company information and real-time stock trades for one month.
+
+    Download CSV: <tag type="download">[stock_data_real_time.zip](https://s3.amazonaws.com/assets.timescale.com/docs/downloads/)</tag>
```

Comment:
Formatting is slightly broken because of Markdown parsing. Pretty sure I know why, I'll put in a fix in the web-documentation repo
![image](https://user-images.githubusercontent.com/26616127/157495312-3495b93f-adda-432a-b6b1-1c95fbfd5073.png)

Reply:
thank you!

Reply:
thank you!